### PR TITLE
Created SecurityBlock tests

### PR DESCRIPTION
**Pull request by D1nviz**
### Test coverage
image
To achieve 100% function coverage, we need to implement the save logic
image

And confirm logic
image

### CODE REVIEW

File: tests/unit/containers/tutor-profile/security-block/SecurityBlock.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+describe('SecurityBlock', () => {
+  beforeEach(() => {
+    render(<SecurityBlock />)
+  })
+
+  it('renders title and description', () => {
+    expect(
+      screen.getByText('editTutor.passwordSecurityTab.title')
```
Comment (Tolik170) [inline tests/unit/containers/tutor-profile/security-block/SecurityBlock.spec.jsx:20]: according to our code convension, move screen.getByText('editTutor.passwordSecurityTab.title') to a variable

File: tests/unit/containers/tutor-profile/security-block/SecurityBlock.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+    expect(
+      screen.getByLabelText('editTutor.passwordSecurityTab.currentPassword')
+    ).toHaveValue('')
+    expect(
+      screen.getByLabelText('editTutor.passwordSecurityTab.newPassword')
+    ).toHaveValue('')
+    expect(
+      screen.getByLabelText('editTutor.passwordSecurityTab.retypePassword')
+    ).toHaveValue('')
```
Comment (Tolik170) [inline tests/unit/containers/tutor-profile/security-block/SecurityBlock.spec.jsx:48]: do you need to check for 3 labels? I mean, maybe 1 is enough

File: tests/unit/containers/tutor-profile/security-block/SecurityBlock.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  it('closes ConfirmDialog when cancel button is clicked', async () => {
+    fireEvent.click(
+      screen.getByText('editTutor.passwordSecurityTab.deactivateAccount')
+    )
+    fireEvent.click(screen.getByText('common.cancel'))
+
+    await waitForElementToBeRemoved(() =>
+      screen.queryByText('editTutor.passwordSecurityTab.deactivateTitle')
+    )
+  })
```
Comment (Tolik170) [inline tests/unit/containers/tutor-profile/security-block/SecurityBlock.spec.jsx:91]: I guess it should expect smth, for example, you can expect for elem with this text to not exist in the document

File: tests/unit/containers/tutor-profile/security-block/SecurityBlock.spec.jsx (hunk 1 above)
Comment (D1nviz) [reply tests/unit/containers/tutor-profile/security-block/SecurityBlock.spec.jsx]: Thank you for conversation! refactored

File: tests/unit/containers/tutor-profile/security-block/SecurityBlock.spec.jsx (hunk 2 above)
Comment (D1nviz) [reply tests/unit/containers/tutor-profile/security-block/SecurityBlock.spec.jsx]: You're right, I changed it

File: tests/unit/containers/tutor-profile/security-block/SecurityBlock.spec.jsx (hunk 3 above)
Comment (D1nviz) [reply tests/unit/containers/tutor-profile/security-block/SecurityBlock.spec.jsx:82]: Thank you, i added 
```
    expect(deactivateTitle).not.toBeInTheDocument()
```

**Review by iva-stasia — APPROVED** [review]

**Review by mhevyk — APPROVED** [review]

**Review by asya982 — APPROVED** [review]

**Review by GeorgeShvab — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**